### ISSUE TITLE
Is this project dead？

### ISSUE THREAD

**Issue by mmm8955405** [ledokku/ledokku#425]
Is this project dead？

**Comment by pradel** [comment]
Not actively maintained but accepting prs

**Comment by weeebdev** [comment]
Any alternatives?

**Comment by gridsystem** [comment]
https://pro.dokku.com

**Comment by weeebdev** [comment]
> https://pro.dokku.com

Wow. No demo? Too expensive

**Comment by gridsystem** [comment]
Dokku is free, I don't blame the developers for creating a premium product people can buy to support it. Talk to @josegonzales he's a nice guy.

**Comment by weeebdev** [comment]
> Dokku is free, I don't blame the developers for creating a premium product people can buy to support it. Talk to @josegonzales he's a nice guy.

No, ofc they have all rights to sell their efforts. I mean, personally for me it's too expensive.

**Comment by mmm8955405** [comment]
Please continue this promising project

**Comment by SaadBazaz** [comment]
This is well-needed, if dokku wants to compete against Caprover.

**Comment by patrickmillerson** [comment]
> Is this project dead？

Looks like
